erpnext/stock/doctype/delivery_trip/delivery_trip.py

```diff
@@ -528,12 +526,14 @@ def get_timesheet():
 				timesheet.save()
 				timesheet.submit()
 
-		frappe.db.set_value("Delivery Trip", trip, "status", "Completed", update_modified=False)
-		frappe.db.set_value("Delivery Trip", trip, "odometer_end_value", odometer_value, update_modified=False)
-		frappe.db.set_value("Delivery Trip", trip, "odometer_end_time", time, update_modified=False)
+		
+		delivery_trip.status = "Completed"
+		delivery_trip.odometer_end_value = odometer_value
+		delivery_trip.odometer_end_time = time
 
 		start_value = frappe.db.get_value("Delivery Trip", trip, "odometer_start_value")
```

Comment:
- Since we have already done a `get_doc` to get the trip at the start of the function, we do not need to do `get_value` again directly use the `odometer_start_value` from it.